curr_scraper/apps/api_caller/models.py

```diff
@@ -11,9 +11,10 @@ class Meta:
 
 class CurrencyRateRecord(models.Model):
     timestamp = models.TimeField()
-    curr = models.ForeignKey(Currency, on_delete=models.PROTECT, related_name='%(class)s_currency')
-    base_curr = models.ForeignKey(Currency, on_delete=models.PROTECT, related_name='%(class)s_base_currency')
-    rate = models.FloatField()
+    curr = models.ForeignKey(Currency, on_delete=models.PROTECT, related_name='currency_name_records_currency')
```

Comment:
Related name is used when you want to get all objects linked to FK model.
For example, in this case we would use it in following scenario:
```
currency_usd = Currency.objects.get(code='USD')
usd_rates = currency_usd.<related_name>
```
So in your proposed solution it would be:
```
usd_rates = currency_usd.currency_name_records_currency
```
which is obviously hard to understand. Much more readable would be:

```
usd_rates = currency_usd.currency_rate_records
usd_rates_base = currency_usd.base_currency_rate_records
```